starter-core/src/test/groovy/io/micronaut/starter/feature/build/gradle/GradleEnterpriseSpec.groovy

```diff
@@ -42,4 +44,45 @@ class GradleEnterpriseSpec extends ApplicationContextSpec implements CommandOutp
         where:
         buildTool << [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN]
     }
+
+    void 'feature gradle-enterprise creates a .mvn/extensions dot xml file'() {
+        when:
+        Map<String, String> output = generate(ApplicationType.DEFAULT, new Options(Language.JAVA, BuildTool.MAVEN), ["gradle-enterprise"])
+        def xml = new XmlParser().parseText(output[".mvn/extensions.xml"])
+
+        then:
+        xml.name() == 'extensions'
+
+        def enterpriseExtension = xml.extension.find { it.artifactId.text() == 'gradle-enterprise-maven-extension' }
+        enterpriseExtension.groupId.text() == 'com.gradle'
+        enterpriseExtension.version.text() ==~ /[\d.]+/ // numbers and fullstops
+        def userDataExtension = xml.extension.find { it.artifactId.text() == 'common-custom-user-data-maven-extension' }
+        userDataExtension.groupId.text() == 'com.gradle'
+        userDataExtension.version.text() ==~ /[\d.]+/ // numbers and fullstops
+    }
+
+    void 'feature gradle-enterprise does not create maven files for #buildTool'() {
+        when:
+        Map<String, String> output = generate(ApplicationType.DEFAULT, new Options(Language.JAVA, buildTool), ["gradle-enterprise"])
+
+        then:
+        output[".mvn/gradle-enterprise.xml"] == null
+        output[".mvn/extensions.xml"] == null
+
+        where:
+        buildTool << [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN]
+    }
+
+    void 'feature gradle-enterprise creates a .mvn/gradle-enterprise dot xml file'() {
+        when:
+        Map<String, String> output = generate(ApplicationType.DEFAULT, new Options(Language.JAVA, BuildTool.MAVEN), ["gradle-enterprise"])
+        def xml = new XmlParser().parseText(output[".mvn/gradle-enterprise.xml"])
+
+        then:
+        xml.name() == 'gradleEnterprise'
+        !xml.server
+        xml.buildScan.termsOfService.url.text() == 'https://gradle.com/terms-of-service'
+        xml.buildScan.termsOfService.accept.text() == 'yes'
```

Comment:
This fails, as it is `true`, not `yes` 🤔

Reply:
Yeah it's not consistent with the Gradle version, I've fixed it by having `GradleEnterpriseConfiguration#getTermsOfServiceAgree` returning a boolean and handling differently in templates